Makefile

```diff
@@ -8,6 +8,7 @@ mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
 test: lint mypy
+	tests/fixtures/populate.py --s3-bucket $(DSS_S3_BUCKET_TEST_FIXTURES) --gs-bucket $(DSS_GS_BUCKET_TEST_FIXTURES)
```

Comment:
The motivation for this change was I added some tests which required additional test fixture data, and the Travis branch build failed because the test fixture data was not available.
I discussed with with Andrey, and he said make needs to setup the necessary data for tests to pass, and populate.py is our way of doing that now, and we will need to make a faster way. It does take a while (about 30 seconds my system).